### PR TITLE
Serialize grid

### PR DESCRIPTION
**Pull request by oelbert**
This PR adds serialization statements  to validate the metric terms generation in fv_grid_tools, fv_grid_utils and fv_control, and bumps the version up to 7.2.6. Code Changes:

- atmosphere.F90: Turning serialization on for fv_init
- fv_control.F90: Added serialization for `init_grid` and `grid_utils_init`
- fv_grid_tools.F90: Added serialization within `init_grid`
- fv_grid_utils.F90: Added serialization within `grid_utils_init`
- FV3/Makefile: Added `tools` to locations of serialization statements
- tests/serialized_test_data_generation/Makefile: Bumped data version to 7.2.6
- check_data_dir.sh: Changed the check for the number of serialized data files to exclude fields with "_master_" in their name, so that fields that only exist on the master rank can be serialized

### CODE REVIEW

File: FV3/atmos_cubed_sphere/tools/fv_grid_tools.F90 [hunk 1]
```diff
@@ -890,11 +920,11 @@ subroutine init_grid(Atm, grid_name, grid_file, npx, npy, npz, ndims, nregions,
           dyc(i,jed+1) = dyc(i,jed)
        end do
 
-
```
Comment (spencerkclark) [inline FV3/atmos_cubed_sphere/tools/fv_grid_tools.F90:893]: Just maybe undo this and the rest of the spacing changes in this file for a cleaner history?

**Review by spencerkclark — APPROVED** [review]
Looks good to me.

File: FV3/atmos_cubed_sphere/tools/fv_grid_tools.F90 (hunk 1 above)
Comment (oelbert) [reply FV3/atmos_cubed_sphere/tools/fv_grid_tools.F90:893]: ok

**Comment by oelbert** [comment]
This PR adds serialization statements to validate the metric terms generation in fv_grid_tools, fv_grid_utils and fv_control, and bumps the version up to 7.2.6. Code Changes:

    atmosphere.F90: Turning serialization on for fv_init
    fv_control.F90: Added serialization for init_grid and grid_utils_init
    fv_grid_tools.F90: Added serialization within init_grid
    fv_grid_utils.F90: Added serialization within grid_utils_init
    FV3/Makefile: Added tools to locations of serialization statements
    tests/serialized_test_data_generation/Makefile: Bumped data version to 7.2.6
    check_data_dir.sh: Changed the check for the number of serialized data files to exclude fields with "master" in their name, so that fields that only exist on the master rank can be serialized

**Pull request merged**